### DISCUSSION TITLE
Extension not work

### DISCUSSION THREAD

**Discussion by Ethanout** [SPGoding/datapack-language-server#1450] [Q&A] [answered]
I've encountered a problem in the latest version of the extension and have exhausted all possible solutions I could think of, yet the problem persists. Below is the error information:
![image](https://github.com/SpyglassMC/Spyglass/assets/113243989/59192add-bf50-489c-85c2-a5b9be224216)
[info.txt](https://github.com/user-attachments/files/16062248/info.txt)
Any ideas on how to fix this?

**Comment by misode** [comment, marked as answer]
It failed to download necessary resources for the language server to function. Is `raw.githubusercontent.com` blocked in your country?

Try using another data source by creating a `spyglass.json` file at the root of your workspace:
```json
{
  "env": {
    "env": "jsDelivr"
  }
}
```

**Comment by AiJYGR** [comment]
[info.log](https://github.com/user-attachments/files/17288161/info.log)
It seems that I also met this problem.
And after trying your method
it shows this:
```
[Error - 13:15:52] Stopping server failed
  Message: Pending response rejected since connection got disposed
  Code: -32097 
``` 
Any other way to solve this? QwQ
